### PR TITLE
fixed possible npe

### PR DESCRIPTION
**Pull request by littleaj**
Fixes possible NPE from #852

### CODE REVIEW

File: agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassVisitor.java [hunk 1]
```diff
@@ -111,9 +111,11 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
             return new PreparedStatementCtorMethodVisitor(access, desc, instrumentationData.getClassName(), name, originalMV, metaData);
         }
 
-        Set<String> foundDesc = nameAndSignatures.get(name);
-        if (desc != null && foundDesc.contains(desc)) {
-            return new PreparedStatementSetMethod(access, desc, instrumentationData.getClassName(), name, originalMV, metaData);
+        if (desc != null) {
```
Comment (dhaval24) [inline agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassVisitor.java]: Curious why would this ever be null? This is basically the parent class signature as far as I know and in order to do instrumentation this signature needs to be passed.

Comment (littleaj) [reply agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassVisitor.java]: that check was already there. so, i can't really speak to that.

Comment (littleaj) [reply agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassVisitor.java]: the point of this PR is the null check on `foundDesc`

Comment (dhaval24) [reply agent/src/main/java/com/microsoft/applicationinsights/agent/internal/agent/sql/PreparedStatementClassVisitor.java]: got it. Looks good.

**Pull request merged**